serverless.yml

```diff
@@ -55,3 +55,15 @@ functions:
     name: ${env:LFW_DATA_TARGET_ENV_NAME}${self:service}-fgsProcess
     handler: lib/functions/fgs-process.handler
     timeout: 60
+  imtdProcess:
+    name: ${env:LFW_DATA_TARGET_ENV_NAME}${self:service}-imtdProcess
+    handler: lib/functions/imtd-process.handler
+    timeout: 120
```

Comment:
We might need to consider what value this should be. I'm assuming this is 120 seconds so it will need to be increased.

Reply:
will deploy as is but refine in testing if needed